src/main/java/frc/robot/RobotContainer.java

```diff
@@ -55,38 +84,9 @@ public RobotContainer() {
    */
   private void configureBindings() {
     
-    // Configure default commands
-    m_robotDrive.setDefaultCommand(
-        // The left stick controls translation of the robot.
-        // Turning is controlled by the X axis of the right stick.
-        new RunCommand(
-            () ->
-                m_robotDrive.drive(
-                    MathUtil.applyDeadband(
-                        -m_driverController.getLeftY(), 0.1),
-                    MathUtil.applyDeadband(
-                        -m_driverController.getLeftX(), 0.1),
-                    MathUtil.applyDeadband(
-                        -m_driverController.getRightX(), 0.1),
-                    true),
-            m_robotDrive));
-
-          if(m_driverController.getLeftStickButtonPressed()){
-            m_robotDrive.zeroHeading();
-          }
-
-          if(m_driverController.getRightStickButtonPressed()){
-            m_robotDrive.setX();
-          }
-
-    new JoystickButton(m_driverController, XboxController.Button.kLeftBumper.value).onTrue(new InstantCommand(() -> intake.toggleIntake(), intake));
```

Comment:
These buttons were removed to cut down on errors related to the lack of arm/intake motor controllers.  please put them back.

Reply:
Added them back.